Error: qdoublematrix4x4.cpp:1075:74: error: 'endl' is not a member of 'Qt'; did you mean 'endl'?
Ref:   https://bugreports.qt.io/browse/QTBUG-82680

diff --git a/examples/location/mapviewer/main.cpp b/examples/location/mapviewer/main.cpp
index ec54b5a..b332de8 100644
--- a/examples/location/mapviewer/main.cpp
+++ b/examples/location/mapviewer/main.cpp
@@ -66,8 +66,8 @@ static bool parseArgs(QStringList& args, QVariantMap& parameters)
 
         if (param.startsWith("--help")) {
             QTextStream out(stdout);
-            out << "Usage: " << Qt::endl;
-            out << "--plugin.<parameter_name> <parameter_value>    -  Sets parameter = value for plugin" << Qt::endl;
+            out << "Usage: " << endl;
+            out << "--plugin.<parameter_name> <parameter_value>    -  Sets parameter = value for plugin" << endl;
             out.flush();
             return true;
         }
diff --git a/examples/location/places/main.cpp b/examples/location/places/main.cpp
index 69d31ae..c677e42 100644
--- a/examples/location/places/main.cpp
+++ b/examples/location/places/main.cpp
@@ -63,8 +63,8 @@ static bool parseArgs(QStringList& args, QVariantMap& parameters)
 
         if (param.startsWith("--help")) {
             QTextStream out(stdout);
-            out << "Usage: " << Qt::endl;
-            out << "--plugin.<parameter_name> <parameter_value>    -  Sets parameter = value for plugin" << Qt::endl;
+            out << "Usage: " << endl;
+            out << "--plugin.<parameter_name> <parameter_value>    -  Sets parameter = value for plugin" << endl;
             out.flush();
             return true;
         }
diff --git a/src/positioning/qdoublematrix4x4.cpp b/src/positioning/qdoublematrix4x4.cpp
index 0bc5646..32cc7f1 100644
--- a/src/positioning/qdoublematrix4x4.cpp
+++ b/src/positioning/qdoublematrix4x4.cpp
@@ -1072,12 +1072,12 @@ QDebug operator<<(QDebug dbg, const QDoubleMatrix4x4 &m)
     }
 
     // Output in row-major order because it is more human-readable.
-    dbg.nospace() << "QDoubleMatrix4x4(type:" << bits.constData() << Qt::endl
+    dbg.nospace() << "QDoubleMatrix4x4(type:" << bits.constData() << endl
         << qSetFieldWidth(10)
-        << m(0, 0) << m(0, 1) << m(0, 2) << m(0, 3) << Qt::endl
-        << m(1, 0) << m(1, 1) << m(1, 2) << m(1, 3) << Qt::endl
-        << m(2, 0) << m(2, 1) << m(2, 2) << m(2, 3) << Qt::endl
-        << m(3, 0) << m(3, 1) << m(3, 2) << m(3, 3) << Qt::endl
+        << m(0, 0) << m(0, 1) << m(0, 2) << m(0, 3) << endl
+        << m(1, 0) << m(1, 1) << m(1, 2) << m(1, 3) << endl
+        << m(2, 0) << m(2, 1) << m(2, 2) << m(2, 3) << endl
+        << m(3, 0) << m(3, 1) << m(3, 2) << m(3, 3) << endl
         << qSetFieldWidth(0) << ')';
     return dbg;
 }
